<html>
<!---------------------------------------------------------------OVERALL FONT AND COLOR---------------------------------------------------------------->
  <head style="background-color:#FDF7DF">

<!-------------------------------------------------------------------FAVICON IMAGE--------------------------------------------------------------------->
<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
      <script src="js/bootstrap.js"></script>

<!--
<script type"text/javascript">
	$(function () {
		$("style[media*='screen']").remove();
		$("link[href*='favicon']").remove();
		//fix heading
		var h1 = $(".firstHeading").text().split("/");
		$(".firstHeading").text(h1[h1.length-1]);
		$("tr:odd").addClass("odd");
	});
	$('link[rel="shortcut icon"]').attr('href','http://openwetware.org/images/6/61/Injectimod_Favicon.png');
</script>
-->
      
<link rel="icon" type="image/png" href="Injectimod_favicon.png">

<!-------------------------------------------------------------CSS FONTS AND SHIT------------------------------------------------------------------------>

<link href="css/bootstrap.css" rel="stylesheet">
<link href="css/biomod_2015.css" rel="stylesheet">
<link href="css/simple-sidebar.css" rel="stylesheet">


<!-------------------------------------------------------------ACTUAL HEADER------------------------------------------------------------->

<div class="navbar navbar-default navbar-fixed-top" role="navigation" style="font-family: KOMIKA; font-size: 125%; background-color: #005c64; border-style: solid; border-color: #9d493f; border-width: 0px 0px 7px 0px">
  <div class="container">
	<div class="navbar-header">
		<button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target=".navbar-collapse">
		  <span class="sr-only">Toggle navigation</span>
		  <span class="icon-bar"></span>
		  <span class="icon-bar"></span>
		  <span class="icon-bar"></span>
		</button>
                <img src="images/Injectimod_bannerlogo.png" style="padding: 3px 15px 0px 10px; width: 191.1; height: 47.32px;">
	</div>
	<div class="navbar-collapse collapse">
	  	  <ul class="nav navbar-nav">
		<li id="home_link"><a href="index.html">Home</a></li>
		<!--<li id="project_link"></li>-->
		<li id="project_link" class="dropdown">
			<a href="#" class="dropdown-toggle" data-toggle="dropdown">Our Project<span class="caret"></span></a>
			<ul class="dropdown-menu" role="menu" style="background-color: #337D83">
				<li><a href="project_background.html" style="background-color: #337D83; color: #ffffff">The Background</a></li>
				<li class="divider"></li>
				<li><a href="project_goals.html" style="background-color: #337D83; color: #ffffff">The Goals</a></li>
			</ul>
		</li>
		<!--<li id="lab_book_link"></li>-->
		<li class="dropdown" id="lab_book_link">
			<a href="#" class="dropdown-toggle" data-toggle="dropdown">Our Lab Book<span class="caret"></span></a>
			<ul class="dropdown-menu" role="menu" style="background-color: #c4928c">
				<li><a href="labbook_page.html" style="background-color: #c4928c; color: #ffffff">Overview</a></li>
				<li class="divider"></li>
				<li><a href="labbook_bioblueprint.html" style="background-color: #c4928c; color: #ffffff">Bio Blueprint</a></li>
                <li class="divider"></li>
				<li style="padding-bottom: 6px"><a href="labbook_proteinproduction.html" style="background-color: #c4928c; color: #8c3425">Protein Production</a></li>
				<li style="padding-bottom: 6px"><a href="labbook_dnadesign.html" style="background-color: #c4928c; color: #ffffff">DNA Design</a></li>
				<li style="padding-bottom: 6px"><a href="labbook_conjugation.html" style="background-color: #c4928c; color: #ffffff">Chemistry Conjugation</a>
				<li style="padding-bottom: 6px"><a href="labbook_imaging.html" style="background-color: #c4928c; color: #ffffff">Imaging</a></li>
				<li style="padding-bottom: 7px"><a href="labbook_kinetics.html" style="background-color: #c4928c; color: #ffffff">Kinetics</a></li>
			</ul>
		</li>
        <li id="team_link"><a href="team_page.html">Our Team</a></li>
		<li id="sponsors_link"><a href="sponsors_page.html">Sponsors</a></li>
	  </ul>
	</div><!--/.nav-collapse -->
  </div>
</div>
  <head>
	
	<script type="text/javascript">
		$('#lab_book_link').addClass('active');
	</script>
      
 <!--------------------------------------------------------------SIDE BAR AND SHIT--------------------------------------------------------------->
      
<body>
  <div class="container" style="text-align:left; padding-top:57px">
    <div id="wrapper">

        <!-- Sidebar -->
        <div id="sidebar-wrapper">
            <ul class="sidebar-nav">
                <li class="sidebar-brand">
                    <a href="#">
                        Protein Production
                    </a>
                </li>
                <li>
                    <a href="#intro" style="color:#fff">INTRODUCTION</a>
                </li>
               
                <li>
                     <a href="#results" style="color:#fff">RESULTS</a>
                    <ul>
                        <li>
                             <a href="#result1" style="color:#fff">Cloning</a>
                        </li>
                        <li>
                             <a href="#result2" style="color:#fff">Protein Expression</a>
                        </li>
                        <li>
                             <a href="#result3" style="color:#fff">Mass Spectroscopy</a>
                        </li>
                        <li>
                             <a href="#result4" style="color:#fff">Size Exclusion Chromatography</a>
                        </li>
                          <li>
                             <a href="#result5" style="color:#fff">Fluoroescent Labelling</a>
                        </li>
                    </ul>
                </li>
                <li>
                    <a href="#conc" style="color:#fff">CONCLUSIONS</a>
                </li>
                <li>
                    <a href="#matsnmets" style="color:#fff">MATERIALS AND METHODS</a>
                </li>

            
            </ul>
        </div>        
    </div>
</div>     
      

<!--------------------------------------------------------------CSS AND SHIT-------------------------------------------------------------------->
      
<style>      
     #proteinprod {
        display:block;
		padding-top: 75px;
        width: 450px;
		height: 321.5px;
		margin-left: auto;
		margin-right: auto;
        background-image: url(images/proteinprodfinal.png);
		background-repeat: no-repeat;
		background-position: bottom;
    }
    
    #proteinprod:hover {
        background-position: top;
    }
    
       p {
        font-weight: bold;
        font-size: 16px;
    }
    
    h6 {
        font-family: KOMIKA;
        font-size: 24px;
        color: #000000;     
    }
    
</style>
      
<!--------------------------------------------------------------CONTENT AND SHIT---------------------------------------------------------------->
      <div class="container" style="padding-top: 75px; padding-bottom: 20px">
        <a id="proteinprod" href="labbook_proteinproduction.html"></a>
</div>       
         <div id="page-content-wrapper">
            <div class="container-fluid">
                <div class="row">
                    <div class="col-lg-12">
                        <h4 id="intro" style="padding-top:75px">Protein Engineering and Production</h4>
                        <p>To build the tip, we needed to make the protein parts that form the complex. Here, we describe how we cloned and produced tip proteins from the bacteria Yersinia pestis (LcrV) and Salmonella typhimurium (SipD) that were engineered to stick specifically to our artificial DNA construction scaffold.</p>

                        </div> 
                        </div>    
                    </div>
             
             
             <br>
            
             <div class="container-fluid">
                <div class="row">
                    <div class="col-lg-12">
                        <h4 id="results" style="padding-top:75px">RESULTS</h4>         
            
                        <h5 id="result1" style="padding-top:20px">Cloning of Tip Protein Sequences</h5>
                       <p>Coding Sequences for LcrV and SipD were ordered as gBLOCKs (IDT) to an expression vector using Gibson assembly and the assembly products were transformed into NEB DH5Î± competent <i> E. coli </i> (High efficiency). Colonies were screened for the presence of the insert by colony PCR and gel electrophoresis (insert length ~1.3kbp). Successful clones were used to inoculate 50mL LB broth and incubated overnight for MiniPreps (Qiagen) Plasmid purification. </p> 
                        <center><img src="images/proteinprod1.jpg">
                            <p>Figure 1.  Gel of tip protein PCR product after gibson assembly.</p>
                        </center>
                    </div>            
                </div>
    
                </div>
                 
                            
                <div class="container-fluid">
                <div class="row">
                    <div class="col-lg-12">
                        <h5 id="result2" style="padding-top:20px">Protein Expression of our Tip Proteins</h5>
                        <p>T7 expression plasmids were transformed into T7 high expression <i> E. coli </i>  and protein expression was induced with IPTG. Samples from different stages of our protein expression and Imobilised Metal Ion Chromatography (IMAC) purification pipeline  were collected for running SDS-PAGE (Figures 2 and 3). The major band corresponded to the expected molecular weights (SipD and LcrV: 39kDa) along with minor bands from contaminants. Protein content of the elutions was quantified by Direct Detect (Millipore) Infrared spectroscopy (SipD: 2.0mg/mL, LcrV: ~12mg/mL).</p>
                            
                      
                        
                        <center><img src="images/ProtProdSipDGel.jpg" width="500">
                            <p>Figure 3.  SDS-PAGE of SipD at different stages of protein expression. </p>
                        </center>
                        
                          <center><img src="images/ProtProdLcrVGel.jpg">
                            <p>Figure 2.  SDS-PAGE of LcrV at different stages of protein expression. </p>
                        </center>
                        
                    </div>            
                </div>
                <div class="row">
                    <div class="col-lg-12">
                        <h5 id="result3" style="padding-top:20px">Confirmation of Protein Identity by Tandem Mass Spectroscopy</h5>
                                                    <p>To confirm that we had indeed purified our target proteins, the major bands from SDS PAGE were extracted from gels and submitted to tandem mass spectroscopy (Bio analytical Mass Spectrometry Facility at UNSW Australia). Spectra were analysed using the MASCOT Peptide Mass Fingerprint algorithm (MATRIX Science) with <a href="images/proteinproductionSipDMS.pdf">SipD</a>
 and <a href="images/proteinproductionLcrVMS.pdf">V-antigen tip protein</a>
 as the primary hits. </p>
                    </div>
                </div> 
                
                
                 <div class="row">
                    <div class="col-lg-12">
                        <h5 id="result4" style="padding-top:20px">Protein Purification using Size Exclusion Chromatography (SEC)</h5>
                            <p>
                            
                    
                                    <p>Elution fractions from IMAC purification were combined and further purified by Size Exclusion Chomatography (SEC). Chromatographs of absorbance at 280 nm reveal that LcrV elutions are largely monodispersed (Figure 4). In contrast about 50% of SipD protein was aggregated, eluted in the void volume of the column. However, most of the remaining protein appeared to be mono-dispersed (Figure 5)..</p>
                                 
                                    
                                         <center>
                                         <img src="images/proteinprodSECLcrV.jpg">
                                            <p>Figure 4. SEC chromatogram for LcrV protein.</p>
                                    </center>
                                 <br>
                                      <center>
                                          <img src="images/proteinprodSECSipD.jpg">
                                            <p>Figure 5. SEC chromatogram for SipD protein.</p>
                                    </center>
                                 

                                    
                                    
                                    
                                    
                  
                            
                            </p>
                    </div>
       </div>
                <div class="row">
                    <div class="col-lg-12">
                        <h5 id="result5" style="padding-top:20px">Fluorescent Labelling of Proteins for Single Molecule Fluorescence</h5>
                       <p>A gBLOCK sequence was ordered from IDT for a LcrV and SipD mutant with an accessible cysteine at the carboxyl terminal. The Gibson assembly reaction and protein expression was successful for the LcrV mutant, but cloning was not successful for the SipD mutant. 
After expressing the LcrV protein, the C-terminal cysteine was labelled with a maleimide Alexa488 fluorophore dye for the purpose of single molecule fluorescence microscopy. Figure 6 is a chromatogram which illustrates our mono-dispersed Alexa488 labelled LcrV protein being eluted at ~15ml.  Figure 7 is a SDS-PAGE gel of our LcrV mutantsâ post-SEC and it protein concentration was determined to be ~ 0.025g.  Figure 8 illustrates the same gel being excited at the wavelength of 495nm to observe the presence of Alexa488 fluorophore dye labelled LcrV proteins. </p>
                            
                        <center><img src="images/proteinprodLabelingSEC.jpg">
<p>Figure 6. SEC chromatogram for Alexa488 fluor dye labelled LcrV protein. </p>
                        </center>
                        
                        <center><img src="images/proteinprodLabellingGelCoom.jpg">
<p>Figure 7. SDS-PAGE gel of LcrV cysteine mutant </p>
                        </center>
                        
                         <center><img src="images/proteinprodLabellingGelFluoro.jpg">
<p>Figure 8. SDS-PAGE gel of LcrV cysteine mutant exited at 495nm emission expression. </p>
                        </center>
                        
                    </div>            
                </div>
              
                
             
             
      <div class="row">
                    <div class="col-lg-12">
                        <h4 id="conc" style="padding-top:75px">Conclusions</h4>
                        <p>We've cloned, expressed, and purified two T3SS Needle Tip protein monomers, confirmed their identity through tanedem mass spectroscopy, and fluorescently labeled one of them.. <br>
                            <h6>Now we've got the basic biomaterials to assemble the tip on a DNA scaffold!!</h6>
                      
                     
                    </div>
 </div>        
             
             
             
             
             
             
             
             
             
             
                
                <div class="row">
                    <div class="col-lg-12">
                        <h4 id="matsnmets" style="padding-top:75px">Materials and Methods</h4>
                <br><br>
                 
                                   <h5>1.	Cloning via Gibson Assembly</h5>

<h6>1.1 SspI digest reaction</h6>
                        <p>A SspI digest mix was prepared using the recipe outlined the table 1.1.1 and the reaction was carried out using the conditions stated in table 1.1.2.</p> <br>
<centre>
<p>Table 1.1.1 SspI digest reaction mix components</p>

<table style="width:100%">
  <tr>
      <td><p>Component</p></td>
      <td><p>Volume (Âµl)/ reaction</p></td>		
      <td><p>Final Concentration</p></td>
  </tr>

  
    <td>Circular LIC_HK vector</td>
    <td>5</td>		
    <td>   ~  1ug/reaction</td>
  </tr>

  <tr>
    <td>NEB buffer</td>
    <td>5</td>		
    <td>1x</td>

  </tr>

    <td>SspI enzyme</td>
    <td>1</td>		
    <td>10 units</td>

  </tr>

    <td>MiliQ water</td>
    <td>39</td>		
    <td>-</td>

  </tr>
    <td>Total volume</td>
    <td>50</td>		
    <td>-</td>

  </tr>
</table>
  </centre>

<br>
      <p>Table 1.1.2. SspI digest reaction conditions</p>

<table style="width:100%">
  <tr>
      <td><p>Step</p></td>
      <td><p>Length of time (minutes)</p></td>		
      <td><p>Temperature  (Â°C)</p></td>
  </tr>

  
    <td>Incubation</td>
    <td>30</td>		
    <td> 37</td>
  </tr>

  <tr>
    <td>NEB buffer</td>
    <td>5</td>		
    <td>1x</td>

  </tr>

    <td>Heat Inactivation</td>
    <td>20</td>		
    <td>65</td>

  </tr>
</table>
<br>

<h6>1.2.	Gel Electrophoresis</h6>
<p>
    Agarose was dissolved to a final concentration of 1% (w/v) in 1x TAE (Biorad) and pre-stained with a drop of RedsafeTM nucleic acid staining solution. The samples were mixed with New Englands Biolabs 6x Gel loading dye and undergone electrophoresis for approximately 30 minutes at 120V. The gel was visualised under UV light post electrophoresis.</p><br>

<h6>1.3.	gBLOCK ligation with digested LIC_HK Vector</h6>

<p>After performing the vector SspI digest reaction, a Gibson assembly reaction mix was made (Table 1.2.1) and incubated at 50 Â°C for 30 minute.</p><br>

<p>Table 1.3.1 Gibson Assembly reaction mix</p>

<table style="width:100%">
  <tr>
      <td><p>Component</p></td>
      <td><p>Volume (Âµl)/ reaction</p></td>		
      <td><p>Final Concentration</p></td>
  </tr>

  
    <td>gBLOCK insert</td>
    <td>1</td>		
    <td>100fmol</td>
  </tr>

  <tr>
    <td>Digested vector</td>
    <td>4</td>		
    <td>~   50ug</td>

  </tr>

    <td>Heat Inactivation</td>
    <td>20</td>		
    <td>65</td>

  </tr>
</tr>

    <td>Gibson Master Mixn</td>
    <td>5</td>		
    <td>1x</td>

  </tr>
</tr>

    <td>Total volume</td>
    <td>10</td>		
    <td>-</td>

  </tr>
</table>

<br>


<h6>1.4.	DH5Î± Transformations </h6>
<p>2Âµl of the Gibson Assembly reaction mix (Table 1.3.1) was cloned into 25Âµl of NEB DH5Î± competent <i> E. coli </i> (High efficiency) by heat shocking at 42 Â°C for 45 seconds. 100 Âµl of NEB SOC Outgrowth medium was added to the transformed cells and incubated at 37 Â°C for approximately an hour. The transformed cells were spread onto Luria Broth (LB) agar containing kanamycin (final concentration of 37Âµg/ Âµl) and incubated at 37Â°C overnight.</p><br>


<h6>1.5.	Colony PCR</h6>

<p>Single colonies from 1.4 was picked and diluted in 50 Âµl of MiliQ water. A master mix (table 1.4.1) was prepared and a PCR reaction was carried out on Applied Biosystems Veriti â 96-well Thermal Cycler using the conditions outlined in table 1.4.2. Once the PCR reaction has been completed, gel electrophoresis was performed. </p>
 <br><br>
<p>Table 1.5.1 Colony PCR master mix</p>


<table style="width:100%">
  <tr>
      <td><p>Component</p></td>
      <td><p>Volume (Âµl)/ reaction</p></td>		
      <td><p>Final Concentration</p></td>
  </tr>

  
    <td>GoTaqÂ® Green Master Mix</td>
    <td>1</td>		
    <td>1x</td>
  </tr>

  <tr>
    <td>T7 forward primer</td>
    <td>2.5</td>		
    <td>1ÂµM</td>

  </tr>

    <td>T7 Reverse primer</td>
    <td>2.5</td>		
    <td>1ÂµM</td>

  </tr>
</tr>

    <td>Picked Colony</td>
    <td>2.5</td>		
    <td>-</td>

  </tr>
</tr>

    <td>MiliQ water</td>
    <td>5</td>		
    <td>-</td>

  </tr>
</tr>

    <td>Total volume</td>
    <td>25</td>		
    <td>-</td>

  </tr>
</table>


<br><br>
<p>Table 1.5.2 Colony PCR reaction conditions</p>

<p></p>

<table style="width:100%">
  <tr>
      <td><p>Step</p></td>
      <td><p>Length of time</p></td>		
      <td><p>Temperature  (Â°C)</p></td>
  </tr>

  
    <td>PCR activation step</td>
    <td>30 seconds</td>		
    <td>98</td>
  </tr>

  <tr>
    <td>Three step cycling: 40 cycles</td>
    <td></td>		
    <td></td>

  </tr>

    <td>Denaturation</td>
    <td>10 seconds</td>		
    <td>98</td>

  </tr>
</tr>

    <td>Annealing</td>
    <td>30 seconds</td>		
    <td>55</td>

  </tr>
</tr>

    <td>Extension</td>
    <td>1 minute</td>		
    <td>72</td>

  </tr>
</tr>

    <td>Final Extension</td>
    <td>10 minutes</td>		
    <td>72</td>

  </tr>
</table>

<br>
<h6>1.6	Plasmid Producton and Purification</h6>

<p>A single positive colony from 1.4 was inoculated into 10ml of LB media and incubated at 37 Â°C overnight. On the following day, once the OD600nm of the culture has reached ~ 1 plasmid DNA purification was performed using the QIAGENÂ® Plasmid Purification kit.<p>


<p>After we have successfully made our genetic constructs, these can be cloned into T7 <i> E. coli </i> bacteria so they can mass produce our proteins!</p>
<br><br>

<h5>2.	Protein expression </h5>



<h6>2.1	Transformation into T7 <i> E. coli </i> bacterial culture</h6>

<p>2Âµl of the purified plasmid (section 2.1) was transformed into 25Âµl of T7 high expression <i> E. coli </i> cells via heat shocking at 42Â°C for 45 seconds. 100 Âµl of NEB SOC Outgrowth medium was added to the transformed cells and incubated at 37 Â°C for approximately an hour. The transformed T7 cells were spread onto Luria Broth (LB) agar containing kanamycin (final concentration of 37Âµg/ Âµl) and incubated at 37Â°C overnight.</p><br>

<h6>2.2	Starter Culture</h6>
<p>A starter culture was created by inoculating a single colony of T7 <i> E. coli </i> into 20ml of LB media with kanamycin (final concentration of 37Âµg/ Âµl). This starter culture was incubated at 37Â°C until it reached an OD600 > 1. </p><br>

<h6>2.3	Large Scale Protein Production </h6>
<p>12 x 1 Litre of LB media was prepared: each litre contains 1 drop of antifoam and a final concentration of 37 Âµg/ Âµl Kanamycin. 1ml of the starter culture (section 2.3) was inoculated per litre of LB media and incubated (37Â°C, 150rpm); the OD600nm was measured every hour until it reached mid-log phase (0.4-0.6). </p>
<p>Once the cultures has reached mid-log phase, a sample was collected for SDS-PAGE. IPTG was added (0.5mM final concentration) to the culture to induce gene expression and incubated (18Â°C, 150rpm) overnight. </p><br>


<h6>2.4.	Protein Harvest</h6>

<p>On the following day, the OD600nm of the induced cultures were recorded and a sample was collected for SDS-PAGE. The cultures undergone centrifugation (4000g, 20minutes) to pellet the cells and the supernatant was removed. The cell pellet was resuspended in 1ml of lysis buffer (table 2.5.1) per OD 600nm per 100ml of culture.</p> <br>

<p>For example:</p>
<li>	Volume of bacterial culture grow up: 12 Litres</li>
<li>	OD 600nm of induced culture: 1.6</li>


<p><img src="https://fbcdn-sphotos-b-a.akamaihd.net/hphotos-ak-xpf1/v/t1.0-9/12042928_10208388191808296_8844674618140199199_n.jpg?oh=dd9787cf5f2dc8373cfcaad7e648d404&oe=56C231C3&__gda__=1455455676_1bb5a782db1b36c4cbc52d95ceb672a0"></p>

       
<p>To obtain the cell lysates, the resuspended mixture was sonicated on the Branson Digital Sonifier using the following settings: 10 minutes at 20% amplitude, 1 second on and 1 second off. 
The cell lysates were ultracentrifuged (13,000g, 30 minutes) to attain the soluble fraction which was then decanted into 15ml falcon tubes. A sample of the cell lysate and soluble fraction was collected for SDS-PAGE.</p>
<br>
<p>Table 2.4 lysis buffer recipe mix</p>

<table style="width:100%">
  <tr>
    <td><p>Component</p></td>		
    <td><p>Final Concentration</p></td>
  </tr>

 <tr>
    <td>Tris-HCI (pH=8)</td>	
    <td>20mM</td>
  </tr>

  <tr>
    <td>NaCI</td>
    <td>150mM</td>		

  </tr>

    <td>EDTA- free Protease inhibitor</td>
    <td>1 tablet</td>	

  </tr>
</tr>

    <td>DNAase</td>
    <td>A dip</td>		

  </tr>

</table>
<br>
<h5>3.	Protein Purification</h5>
<p>After we have extracted our protein from the bacteria, we now have to purify it! We aim to do this using histidine tag affinity chromatography and size exclusion chromatography</p>
<br>
<h6>3.1.	Histidine tag affinity chromatography </h6>
<p>1ml Profinity TM IMAC Ni-Charged resin was transferred into a gravity column; the resin and column was washed with three column volumes of MiliQ water and equilibrated with three column volumes of Wash Buffer (table 3.1.1). After the equilibration step, the soluble fraction was transferred into the gravity column and incubated on a slow rotor for ~2 hours at 4Â°C. Once the incubation step has completed, unbounded material was collected in the flow through and washed with three column volumes of Wash Buffer. Samples of unbounded and washed material were collected for SDS-PAGE.</p>

<p>The proteins were subsequently eluted with three column volumes of the elution buffer (table 3.1.2) and samples were also collected for SDS-PAGE. The eluted proteins were concentrated to ~ 2mg/ml via centrifugation (2000g, 5 minutes) in a 10,000 MW membrane concentrator. This concentrated sample will be used for Size Exclusion Chromatography.</p>
<br>
<p>Table 3.1.1 Wash buffer recipe mix</p>

<table style="width:100%">
  <tr>
    <td><p>Component</p></td>		
    <td><p>Final Concentration</p></td>
  </tr>

 <tr>
    <td>Tris-HCI (pH=8)</td>	
    <td>20mM</td>
  </tr>

  <tr>
    <td>NaCI</td>
    <td>150mM</td>		

  </tr>

    <td>EDTA- free Protease inhibitor</td>
    <td>1 tablet</td>	

  </tr>
</tr>

    <td>Imidazole</td>
    <td>20mM</td>		

  </tr>

</table>
<br>
<p>Table 3.1.2 Elution buffer recipe mix</p>

<table style="width:100%">
  <tr>
    <td><p>Component</p></td>		
    <td><p>Final Concentration</p></td>
  </tr>

 <tr>
    <td>Tris-HCI (pH=8)</td>	
    <td>20mM</td>
  </tr>

  <tr>
    <td>NaCI</td>
    <td>150mM</td>		

  </tr>

    <td>EDTA- free Protease inhibitor</td>
    <td>1 tablet</td>	

  </tr>
</tr>

    <td>Imidazole</td>
    <td>500mM</td>		

  </tr>

</table>

<br>
<h6>3.2.	Size Exclusion chromatography (SEC)</h6>
<p>Size exclusion chromatography was performed using the Superdex 200 10/300 GL column. The column was equilibrated with the SEC buffer (table 3.2.1) at a flow rate of 0.5ml/minute for an hour. Approximately 500Âµl of the concentrated protein (section 3.1) was loaded and purified using the conditions as described in table 3.2.2.</p>
<br>
<p>Table 3.2.1 SEC buffer recipe mix</p>

<table style="width:100%">
  <tr>
    <td><p>Component</p></td>		
    <td><p>Final Concentration</p></td>
  </tr>

 <tr>
    <td>Tris-HCI (pH=8)</td>	
    <td>20mM</td>
  </tr>

  <tr>
    <td>NaCI</td>
    <td>150mM</td>		

  </tr>

    <td>EDTA- free Protease inhibitor</td>
    <td>1 tablet</td>	

</table>
<br>
<p>Table 3.2.2 Conditions used for SEC on Superdex 200 10/300 GL column</p>
<table style="width:50%">
  <tr>
    <td><p>Parameters</p></td>		
    <td><p>Value</p></td>
  </tr>

 <tr>
    <td>Flow rate</td>	
    <td>0.5ml/minutes</td>
  </tr>

  <tr>
    <td>Maximum Pressure</td>
    <td>1.5 MPa</td>		

  </tr>

    <td>Method / Run length</td>
    <td>1.25 column volumes</td>	

</table>
<br>

<h6>3.3.	Protein storage</h6>
10% glycerol was added to the pure fractions of the protein and snap-freeze in liquid nitrogen. These samples were stored at -80Â°C and used for the following experiments: bio- layer interferometry, small-angle X-ray scattering (SAXS), atomic force microscopy and electron microscopy.</p><br>
<h6>3.4.	SDS-PAGE</h6>
<p>Samples collected from the different steps of protein expression and purification were mixed with NuPAGE LDS sample buffer (4X) and loaded onto a NuPAGE 4-12% Bis-Tris gel. Electrophoresis was performed at 200 Volts for 30 minutes. The gel was stained with SimplyBlueTM SafeStain for an hour post electrophoresis and de-stained with MiliQ water overnight. The gel was visualised under visual light</p><br>


<h5>4.	Protein Labelling</h5>
<h6>4.1.	Alexa488 protein labelling</h6>
<p>400Âµl of 50% slurry of Profinity TM IMAC Ni-Charged resin was equilibrated with 3x3 column volumes of lysis buffer (Table 2.5.1).</p>
<p>Following from the protein harvest (section 2.5) using the LcrV_Cysteine mutants, the equilibrated 50% slurry along with other components were added to the soluble fraction within a 15ml falcon tube as detailed in table 4.1.1. Once added, the soluble were incubated on a slow rotor at 4Â°C for ~2 hours. </p><br>

<p>Table 4.1.1 Components added to LcrV_Cysteine mutant soluble fraction</p>

<table style="width:50%">
  <tr>
    <td><p>Component</p></td>		
    <td><p>Final Concentration</p></td>
  </tr>

 <tr>
    <td>TCEP</td>	
    <td>200ÂµM</td>
  </tr>

  <tr>
    <td>Imidazole</td>
    <td>10mM</td>		

  </tr>

    <td>Equilibrated Ni-Charged resin</td>
    <td>400Âµl (50%)</td>	

</table>
<br>
<p>After this incubation step, the soluble fraction was transferred into Micro Bio-SpinÂ® Chromatography Columns and washed with 20 column volumes of Wash Buffer (Table 4.1.1) along with five column volumes of rinse buffer (Table 4.1.2). The Ni-charged resin was resuspended to 50% slurry with the rinse buffer and protein concentration was quantified via SDS-PAGE (section 3.3) against known concentrations of BSA. </p>
<br>
<p>Table 4.1.2 Rinse buffer recipe mix</p>
<table style="width:50%">
  <tr>
    <td><p>Component<p6></td>		
    <td><p>Final Concentration</p></td>
  </tr>

 <tr>
    <td>Tris-HCI (pH=8)</td>	
    <td>20mM</td>
  </tr>

  <tr>
    <td>Imidazole</td>
    <td>10mM</td>		

  </tr>

    <td>NaCI</td>
    <td>300mM</td>	

</table>
<br>

<p>In an Eppendorf tube, the 50% slurry was incubated with TCEP (200ÂµM) at room temperature for 30 minutes and 5X molar excess of Alexa FluorÂ® 488 dye was added. This mixture was covered in aluminium foil and incubated at 4Â°C overnight. </p>
<br>
<h6>4.2.	Labelling purification</h6>

<p>After the incubation, the resin was transferred to a Chromatography Column and washed with 20 column volumes of Wash Buffer (Table 3.1.1) for the removal of unbound excess fluorophores. The labelled proteins were then eluted in 3 x column volume fractions using the elution buffer (Table 3.1.2). </p>

<p>SEC was performed as outlined in section 3.2 to separate remaining excess fluorophore from the labelled protein. Monomeric protein concentration was quantified on SDS-PAGE (section 3.3) against known concentrations of BSA while concentration of labelled protein was determined via UV-visible scan on the NanoDrop 1000 Spectrophometer (1mm path length).</p><br>

<p>The labelling efficiency was calculated using the formula:</p>
<img src= "https://scontent.xx.fbcdn.net/hphotos-xtf1/v/t1.0-9/12039593_10208388587858197_6371267453771134825_n.jpg?oh=af9d023aae92a5c5e7acc98b3c3be39c&oe=56C02527">
<p>Ax=absorbance value of dye at absorption maximum wavelength.E dye= extinction coefficient of the dye or reagent at the absorption maximum wavelength </p><br>

<h6>4.3.	Alexa 488 fluorophore labelled Protein storage</h6>
<p>10% glycerol was added to the pure fractions of the labelled protein, snap-freeze in liquid nitrogen and stored at -80Â°C. The labelled proteins will be used for Single molecule fluorescent microscopy.</p>

<br><br>
               
<p> </p>
<p> </p>
                        
<!--------------------------------------------------------------ACTUAL FOOTER------------------------------------------------------------------->

    <div class="footer">
        <div class="container">
	     <div style="height: 5px;"></div>
        <p style="font-size: 80%; font-family: GUDEA, Arial, serif; color: #ffffff;">Contact: <a id="footer_link" href="mailto:BIOMODaustralia@gmail.com">BIOMODaustralia@gmail.com</a> | &copy;2015 Team Injectimod. All rights reserved. This team is supported by the <a id="footer_link" href="http://www.unsw.edu.au">University of New South Wales</a> and <a id="footer_link" href="http://www.victorchang.edu.au">Victor Chang Cardiac Research Institute</a>
        </div>
    </div>
      

</html>
